website/docs/terminology.md

```diff
@@ -293,24 +237,74 @@ A program on the [host] that [verifies] a [receipt].
 
 ### Verify
 
-[verifying]: #verify
-[verifies]: #verify
-
 Check that the [receipt] is valid.
 Verifying [segment] receipts requires checking that the [seal] is valid and was generated by the expected [ImageID].
 Verifying [session] receipts requires checking the validity of each [segment] and checking that the initial/final memory snapshots for each segment align appropriately.
 
 ### Zero-Knowledge Virtual Machine (zkVM)
 
-[zkVM]: #zero-knowledge-virtual-machine-zkvm
-
 A virtual machine that runs trusted code and generates proofs.
 RISC Zero's zkVM implements the RISC-V instruction set architecture and uses a [STARK]-based proof system.
 
 [About STARKs]: ./reference-docs/about-starks.md
+[Arithmetic Circuits]: /reference-docs/about-arithmetic-circuits
+[assumption]: #assumption
+[assumptions]: #assumption
 [cargo risczero]: https://docs.rs/crate/cargo-risczero/latest
+[circuit]: #circuit
+[clock cycles]: #clock-cycles
+[commit]: #commit
+[compose]: #composition
+[composition]: #composition
+[conditional]: #conditional
+[continuations]: #continuations
+[Continuations study club]: https://www.youtube.com/watch?v=v4HIwaqmIxk&list=PLcPzhUaCxlCirUkJY0ltpjdtzWcz5U_6y&index=1
+[Continuations blog]: https://www.risczero.com/news/continuations
+[control ID]: #control-id
+[control root]: #control-root
+[deterministic-builds]: #deterministic-builds
+[ELF binary]: #elf-binary
+[env::commit()]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/guest/env/fn.commit.html
+[execute]: #execute
+[executes]: #execute
+[execution]: #execute
+[executed]: #executor
+[executor]: #executor
+[Executor documentation]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/struct.Executor.html
+[execution trace]: #execution-trace
+[Groth16]: #groth16
+[Groth16 Circuit]: #groth16-circuit
+[guest]: #guest-program
+[guest program]: #guest-program
+[host]: #host
+[imageID]: #image-id
 [ImageID documentation]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/struct.SystemState.html
 [ImageID excerpt from Study Club]: https://www.youtube.com/watch?v=QwzrBHHkzFE&list=PLcPzhUaCxlCirUkJY0ltpjdtzWcz5U_6y&index=4
+[journal]: #journal
+[method]: #method
+[prove]: #prove
+[proves]: #prove
+[proven]: #prover
+[prover]: #prover
```

Comment:
This (and other examples) can be collapsed. In markdown you can do things like `[proven][prover]` and `[proven][prover]`, so that we only need a single entry in the list of links.

Reply:
tried to implement this suggestion but it seems to break everything. 
landing this as is because i have a follow-up PR i want to open